releases/release-5.3.1.md

```diff
@@ -0,0 +1,169 @@
+---
+title: TiDB 5.3.1 Release Notes
+---
+
+# TiDB 5.3.1 Release Notes
+
+Release Date: xx, 2022
+
+TiDB version: 5.3.1
+
+## Compatibility changes
+
+- Tools
+
+    + TiDB Lightning
+    
+         - Decreate the default region max-keys from 1_440_000 to 1_280_000, to avoid too many empty Regions after data import [#30018](https://github.com/pingcap/tidb/issues/30018)
```

Comment:
```suggestion
         - Decreate the default value of the region max-keys from 1_440_000 to 1_280_000, to avoid too many empty Regions after data import [#30018](https://github.com/pingcap/tidb/issues/30018)
```